### ISSUE TITLE
BERTAug affects Proper Noun

### ISSUE THREAD

**Issue by memahesh** [makcedward/nlpaug#2]
Hi, 

I have been using your BERTAug. for Text Augmentation. It works fine on a lot of tasks but it starts messing up the Proper Nouns.

![image](https://user-images.githubusercontent.com/19503624/58860955-f39a4400-86ca-11e9-8ae3-50602db0ed48.png)

Is there any fix to this ?

**Comment by makcedward** [comment]
@memahesh 
There is a bug. The expected output should concatenate tokenizer word. It will be fixed in next release and the expected output should be not include "#". 

For proper nouns case, it will treat as general word. In other words, "Mahesh" treats as a word and it may replace by other words. Here is one of the possible outcome.
`bhesh is a good person`
If you prefer to reserve the proper noun, you may leverage include it in stopwords list and passing to augmenter. Here is the example. 

```
stopwords = ['mahesh']
aug = naw.BertAug(action=Action.SUBSTITUTE, stopwords=stopwords)
```

**Comment by makcedward** [comment]
Fixed in BETA release. You may try again. It will be included 0.0.4 release.

**Comment by anvitha-jain** [comment]
AttributeError: module 'nlpaug.augmenter.word' has no attribute 'BertAug'
why?
